src/frontends/onnx/frontend/src/op/gemm.cpp

```diff
@@ -30,10 +30,7 @@ OutputVector gemm(const Node& node) {
     }
 
     const auto alpha = node.get_attribute_value<float>("alpha", 1);
-    const auto beta = node.get_attribute_value<float>("beta", 1);
-
-    const auto alpha_node = default_opset::Constant::create(input_b.get_element_type(), Shape{}, {alpha});
-    const auto beta_node = default_opset::Constant::create(input_c.get_element_type(), Shape{}, {beta});
+    const auto beta_node = node.get_attribute_as_constant<float>("beta", 1, input_c.get_element_type());
```

Comment:
I think you can do the same for 'beta' from set_6::gemm

Reply:
Done